pywinauto/timings.py

```diff
@@ -451,11 +451,7 @@ def wait_until_passes(timeout,
                 time.sleep(min(retry_interval, time_left))
 
             else:
-                # Raise a TimeoutError - and put the original exception
-                # inside it
-                err = TimeoutError()
-                err.original_exception = e
-                raise err
+                raise e

```

Comment:
Why have you changed this logic without discussing it first? What was wrong with it? Now it breaks backward compatibility.

Reply:
Well, nevermind. Since tests are good, let's consider backward compatibility is not broken.